forge-gui/res/cardsfolder/upcoming/burning_sun_cavalry.txt

```diff
@@ -0,0 +1,9 @@
+Name:Burning Sun Cavalry
+ManaCost:1 R
+Types:Creature Human Knight
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | IsPresent$ Dinosaur.YouCtrl | Execute$ TrigPump | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks or blocks while you control a Dinosaur, CARDNAME gets +1/+1 until end of turn.
+T:Mode$ Blocks | ValidCard$ Card.Self | IsPresent$ Dinosaur.YouCtrl | Execute$ TrigPump | Secondary$ True | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks or blocks while you control a Dinosaur, CARDNAME gets +1/+1 until end of turn.
```

Comment:
the fact you use NoResolvingCheck$ correctly above but not here makes me wonder if you understand "intervening if" 🤔